driver/kubernetes/kubernetes.go

```diff
@@ -363,15 +370,37 @@ func (k *Driver) deleteJob(name string) error {
 }
 
 func generateNameTemplate(op *driver.Operation) string {
```

Comment:
It would be helpful to document what size string this function aims to generate. Also, is it really generating a "name template"??

Reply:
> Also, is it really generating a "name template"??

Yes, in the sense that it's used by `metav1.ObjectMeta.GenerateName`; I've added a comment to clarify this.